uctMaths/competition/compadmin.py

```diff
@@ -630,6 +634,51 @@ def export_awards(request):
     output_workbook.save(response)
     return response
 
+def makeCertificates(students, assigned_school):
+    
+    certPath = "../../Certificates/"
+
+    if len(students) > 0:
+        awardCerts = {"G": "goldTemplate.docx", "M": "meritTemplate.docx", None: "participationTemplate.docx", "MOX": "meritOxfordTemplate.docx", "OX": "oxfordTemplate"}
+
+        try:
+            os.mkdir(str(students[0].school).replace(" ", "_") + "_Certificates")
+    
+        finally:
+            certs = []
+            filename = str(str(students[0].school).replace(" ", "_") + "_Certificates")
```

Comment:
I think you should test what happens when there is a unicode character in a school name like ë

Reply:
Yeah, this seems like an issue. Would the best way be to simply remove the unicode characters, or is there a better way to do this?

Reply:
Instead of `str` use `unicode`, although if that still breaks in other places, it's probably best just to remove the unicode characters.